scout/server/blueprints/dashboard/controllers.py

```diff
@@ -43,20 +43,43 @@ def get_dashboard_info(adapter, institute_id=None, slice_query=None):
     data['analysis_types'] = get_analysis_types(adapter, total_cases,
                                 institute_id=institute_id, slice_query=slice_query)
 
+    # VARIANTS
+
+    # Here we disregard any slice query.
+    general_info = get_general_case_info(adapter, institute_id=institute_id)
+
+    total_cases = general_info['total_cases']
     # Fetch variant information
     LOG.info("Fetch sanger variants")
-    validation_query = {'sanger_ordered': True}
+    order_query = {'sanger_ordered': True}
```

Comment:
Would it be possible here to have a query which combines 'sanger_ordered', and 'validation' in ['True positive', 'False positive]? Then you could loop on the results just once instead of having 3 different loops.

Reply:
But with your event query instead, obviously!